kubernetes/geo-deploy.yaml

```diff
@@ -25,10 +25,10 @@ spec:
           resources:
             limits:
               memory: "128Mi"
-              cpu: "500m"
+              cpu: "250m"
             requests:
               memory: "128Mi"
-              cpu: "500m"
+              cpu: "250m"
```

Comment:
Why do we lower the reqested cpu value?

Reply:
Azure student subscription is not enough for 3x500m. 250m is enough for our needs, even 100 would be enough